README.md

```diff
@@ -152,20 +152,11 @@ The tails of the MultiCombiner assembly will give access to the results for each
 <b>Things to Watch Out For</b>
 <p>The ability to run arbitrary aggregators over a single stream is pretty useful, but there’s nothing magical going on in the background. Each aggregator used in a  MultiCombiner emits roughly the same number of tuples as if it were being used on its own. Because the sorting involved in the shuffle is an n*log(n) operation, shuffling the output of many aggregators all at once is less efficient than shuffling their outputs separately. This is usually not an issue because of the time saved reading the data set only once, but may matter if the number of tuples being shuffled is much larger than the data being read from disk. Additionally, each aggregator must keep its own map in memory for its combiner. Because of the additional memory pressure, combining for many aggregators can potentially be less efficient. All of that being said, we've seen significant performance increases for every set of aggregation operations we've merged using this tool.
 
-Download
-====
-You can either build cascading_ext from source as described below, or pull the latest jar from the Liveramp Maven repository:
-
-```xml
-
-<repository>
-  <id>repository.liveramp.com</id>
-  <name>liveramp-repositories</name>
-  <url>http://repository.liveramp.com/artifactory/liveramp-repositories</url>
-</repository>
-```
+## Download
 
-Version 0.1 is built off of Cloudera Hadoop 3, (CDH3u3).  The current snapshot version (1.6) is built against CDH4.1.2.  Both are available via Maven:
+You can either build cascading_ext from source as described below, or pull the
+latest jar from Maven Central. Version 0.1 is built off of Cloudera Hadoop 3,
```

Comment:
Is this true?  I don't see it deployed there yet http://repo1.maven.org/maven2/com/liveramp/

Reply:
no, you are right, it's not there yet.
What do you think about downgrading all this to snapshot style releasing? In the future I want to make another pass here to actually release things. I can just suppress the doc guidance for now.

Reply:
Yeah I'm fine just having snapshots for now, I just want to make sure we don't tell people it's available in central before it is.